tcf_website/templates/search/search.html

```diff
@@ -14,6 +14,14 @@ <h1>Search: {{query}}</h1>
 
     <hr>
 
+    {% if courses_error %}
+    <div class="container">
+        <h4>Searching courses is down right now!</h4>
+        <li>{{courses_error}}</li>
+    </div>
+    {% endif %}
```

Comment:
Should there be an `{% else %}` to display courses?

Reply:
true! that makes more sense